tests/requirements-310.txt

```
@@ -8,8 +8,8 @@ fastapi>=0.92.0
 flask>=2.3.2
 markupsafe>=2.1.0
 grpcio>=1.37.1
-google-cloud-pubsub<=2.1.0
-google-cloud-storage>=1.24.0
+google-cloud-pubsub<=2.23.0
```

If we can support the latest version, set a lower-bound version instead of an upper-bound limit.  That way, we can always test and guarantee the most updated version.

```suggestion
google-cloud-pubsub>=2.1.0
```

Please, adapt all other files for this change.

---

Sure, updating now.